react_excercise/got-app/src/Api.js

```diff
@@ -0,0 +1,80 @@
+import axios from 'axios';
+import levenshteinDistance from './LevenshteinDistance';
+
+const url = 'https://thronesapi.com/api/v2/Characters';
+
+async function fetchData() {
+    try {
+        const response = await axios.get(url);
+        return response.data;
+    } catch (error) {
+        console.error('Error fetching data from API');
+        throw error;
+    }
+}
+
+async function searchForCharacters(input) {
```

Comment:
Minor -It would be better to move 'searchForCharacters' and 'getHouseMembers' into their respective controller functions. Since they are working on the data they got from api and not actually fetching it.

Reply:
I seperated out those into seperate controllers.  Makes it cleaner.